src/main/java/org/codehaus/mojo/versions/api/VersionsHelper.java

```diff
@@ -225,14 +225,15 @@ PluginUpdatesDetails lookupPluginUpdates( Plugin plugin, boolean allowSnapshots
      * @param propertyDefinitions Any extra hints about properties.
      * @param includeProperties A comma separated list of properties to include.
      * @param excludeProperties A comma separated list of properties to exclude.
+     * @param includeParent whether to include parent POMs
      * @param autoLinkItems whether to automatically infer associations
      * @return a map of {@link org.codehaus.mojo.versions.api.PropertyVersions} values keyed by
      *         {@link org.codehaus.mojo.versions.Property} instances.
      * @throws MojoExecutionException if something goes wrong.
      */
     Map<Property, PropertyVersions> getVersionPropertiesMap( MavenProject project, Property[] propertyDefinitions,
                                                              String includeProperties, String excludeProperties,
-                                                             boolean autoLinkItems )
+                                                             boolean includeParent, boolean autoLinkItems )
```

Comment:
What do you think to introduce class like `VersionPropertiesMapRequest` with builder instead of parameters list. 
When we have such many parameters with the same type is can be error prone.

Reply:
> Why you removed `it-property-updates-report-issue-684-001` test?

If you look at the test, especially at the goal being tested:

```properties
invoker.goals = ${project.groupId}:${project.artifactId}:${project.version}:plugin-updates-report
```

It looks like it was a duplicated it-plugin-updates-report-issue-684-001, with no changes whatsoever...

Reply:
> What do you think to introduce class like VersionPropertiesMapRequest with builder instead of parameters list.
When we have such many parameters with the same type is can be error prone.

Ok, fair enough.

Reply:
Done.